src/utils.py

```diff
@@ -101,6 +121,9 @@ def submit_updates(
 
     latest_release = collection_repo.latest_release().tag_name
 
+    if collection_name == "prefect":
```

Comment:
Perhaps we can just do if latest_release is not prefixed with v, add v

Reply:
that would also work